src/sentry/static/sentry/app/components/versionHoverCard.jsx

```diff
@@ -115,16 +117,16 @@ const VersionHoverCard = createReactClass({
     let {orgId} = this.props;
     return {
       body: (
-        <div className="version-hovercard blankslate m-a-0 p-x-1 p-y-1 align-center">
+        <Box p={2} className="align-center">
```

Comment:
This changes the padding by a couple pixels, but increases cohesion with the react-grid layout.

Reply:
Would you be able to share a screenshot of the diference?

Reply:
### Before 
![versioncard before](https://user-images.githubusercontent.com/24086/45628404-8898d680-ba83-11e8-84fd-ada3bde0753e.png)

### After

![screen shot 2018-09-17 at 10 10 57 am](https://user-images.githubusercontent.com/24086/45628423-93ec0200-ba83-11e8-9cb2-513f2141764b.png)